ansible/roles/controller/tasks/main.yml

```diff
@@ -0,0 +1,37 @@
+---
+- name: Install openstack build dependencies
+  package:
+    name: "{{ item }}"
+    state: latest
+  become: true
+  with_items:
+    - centos-release-openstack-ocata
+
+- name: Install build dependencies
+  package:
+    name: "{{ item }}"
+    state: latest
+  become: true
+  with_items:
+    - python2-requests
+    - git
+
+- include_role:
```

Comment:
Can we use list of roles here instead of using ```include role``` twice?

Reply:
+1